java/http/vertx/java-http-vertx-producer/src/main/java/ProducerApp.java

```diff
@@ -3,30 +3,29 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import java.util.Map;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.TimeUnit;
-
-import io.jaegertracing.Configuration;
-import io.opentracing.Tracer;
-import io.opentracing.util.GlobalTracer;
-import org.apache.logging.log4j.Logger;
-import org.apache.logging.log4j.LogManager;
-
 import io.vertx.config.ConfigRetriever;
 import io.vertx.config.ConfigRetrieverOptions;
 import io.vertx.config.ConfigStoreOptions;
 import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
 import io.vertx.core.json.JsonObject;
+import io.vertx.tracing.opentelemetry.OpenTelemetryOptions;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 public final class ProducerApp {
-    
-    private static final Logger log = LogManager.getLogger(ProducerApp.class);
 
-    private static String deploymentId;
+    private static final Logger log = LogManager.getLogger(ProducerApp.class);
 
     public static void main(String[] args) throws Exception {
-        Vertx vertx = Vertx.vertx();
+
+        VertxOptions vertxOptions = new VertxOptions();
+        vertxOptions.setTracingOptions(new OpenTelemetryOptions());
+        Vertx vertx = Vertx.vertx(vertxOptions);
```

Comment:
Ditto as above, OpenTelemetry options to be set only if the user sets the tracing system.

Reply:
TracingSystem deployed so tracing is only  enabled of `openTelemetry` is used with `TracingSystem` envVar.